tests/sync/test_admin_status.py

```
@@ -0,0 +1,355 @@
+        with self.assertRaises(requests.HTTPError):
+            SonicFax().check_health()
+
+    @mock.patch.dict(os.environ, _SONIC_ENV)
+    @mock.patch(_SESSION_GET)
+    @mock.patch(_LOGIN, return_value={"c": "v"})
+    def test_check_health_raises_when_bounced_to_login(self, mock_login, mock_get):
+        """A 200 that is really the login form must not count as healthy."""
+        from fighthealthinsurance.fax_utils import SonicFax
+
+        mock_get.return_value = _members_response(text="Please Member Login")
+        with self.assertRaises(Exception):
```

Ruff (0.15.17)

[warning] Do not assert blind exception: `Exception`

(B017)